### PR TITLE
onSubDomainChange event

### PR DESCRIPTION
**Pull request by valerii-cognite**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-83.surge.sh

File: src/components/DataProvider/index.js [hunk 1]
```diff
@@ -315,6 +325,7 @@ DataProvider.propTypes = {
   children: PropTypes.node.isRequired,
   defaultLoader: PropTypes.func,
   series: seriesPropType.isRequired,
+  onSubDomainChanged: PropTypes.func,
```
Comment (evancharlton) [inline src/components/DataProvider/index.js]: Could you add a `// (args) => null` comment? I don't know what this callback gets back.

**Comment by evancharlton** [comment]
Feel free to submit this whenever that comment is added.

File: src/components/DataProvider/index.js (hunk 1 above)
Comment (valerii-cognite) [reply src/components/DataProvider/index.js]: done

**Pull request merged**